QtCollider/primitives/prim_misc.cpp

```diff
@@ -325,32 +329,32 @@ QC_LANG_PRIMITIVE( Qt_SetAppMenus, 1, PyrSlot *r, PyrSlot *a, VMGlobals *g )
       }
     }
   }
-  
+
   return errNone;
 }
 
 QC_LANG_PRIMITIVE( QView_AddActionToView, 3, PyrSlot *r, PyrSlot *a, VMGlobals *g )
 {
   if( !QcApplication::compareThread() ) return QtCollider::wrongThreadError();
-  
+
   QObjectProxy* widgetObj = TypeCodec<QObjectProxy*>::safeRead(a);
   QObjectProxy* actionObj = TypeCodec<QObjectProxy*>::safeRead(a + 1);
   QObjectProxy* beforeObj = TypeCodec<QObjectProxy*>::safeRead(a + 2);
-  
+
   if (widgetObj && actionObj) {
     QWidget* widget = qobject_cast<QWidget*>(widgetObj->object());
     QAction* action = qobject_cast<QAction*>(actionObj->object());
     QAction* beforeAction = beforeObj ? qobject_cast<QAction*>(beforeObj->object()) : 0;
-    
+
     if (!beforeAction) {
       PyrSlot* indexArg = a + 2;
-    
+
       if (NotNil(indexArg)) {
         if (IsInt(indexArg)) {
           int index = std::max(slotRawInt(indexArg), 0);
-          
+
           auto actions = widget->actions();
-          
+
```

Comment:
You really need to change your editor settings to stop stripping trailing whitespace when you edit. We are already planning to do this project-wide, so all this does is great git blame and git diff noise.

Reply:
ok